src/js_native_api_v8.cc

```diff
@@ -1483,6 +1483,8 @@ napi_status napi_create_string_latin1(napi_env env,
                                       size_t length,
                                       napi_value* result) {
   CHECK_ENV(env);
+  if (length > 0)
+      CHECK_ARG(env, str);
```

Comment:
The naming of these macros is really unfortunate. Everywhere else, `CHECK_*` generally implies an assert.